config/config_mgmt.py

```diff
@@ -198,6 +200,12 @@ def readConfigDB(self):
 
         return
 
+    def sortData(self, data):
```

Comment:
@tahmed-dev : We do not need it now, https://github.com/Azure/sonic-utilities/pull/1246

I thought --print-data for sonic-cfggen must print sorted value, but if that is not true, We can remove this function all together now.

Reply:
@praveen-li I was not aware of [PR:1246](https://github.com/Azure/sonic-utilities/pull/1246). Closing this one in favor of [PR:1246](https://github.com/Azure/sonic-utilities/pull/1246)